diff --git a/web/wp-config.php b/web/wp-config.php
index 81df54d..502320e 100644
--- a/web/wp-config.php
+++ b/web/wp-config.php
@@ -5,6 +5,7 @@
  * This file is required in the root directory so WordPress can find it.
  * WP is hardcoded to look in its own directory or one directory up for wp-config.php.
  */
-require_once dirname(__DIR__) . '/vendor/autoload.php';
-require_once dirname(__DIR__) . '/config/application.php';
+
+require_once dirname(__DIR__) . '/bedrock/vendor/autoload.php';
+require_once dirname(__DIR__) . '/bedrock/config/application.php';
 require_once ABSPATH . 'wp-settings.php';
